### PR TITLE
[WIP] Notify roslint results to PR webpage

### PR DESCRIPTION
**Pull request by wkentaro**
@furushchev
Could you please notify to PR in notify_roslint_results.py ?

Modified:
	 travis.sh
Added:
	 added_lines.sh
	 notify_roslint_results.py

### CODE REVIEW

**Comment by wkentaro** [comment]
```
ERROR: cannot download default sources list from:

https://raw.github.com/ros/rosdistro/master/rosdep/sources.list.d/20-default.list

Website may be down.

```

**Comment by wkentaro** [comment]
sample output of diff.patch and roslint_result.txt.
https://gist.github.com/ba16ad862f380d2e689e

**Comment by wkentaro** [comment]
```
['/home/wkentaro/ros/indigo/src/jsk-ros-pkg/jsk_common/jsk_topic_tools/src/hz_measure_nodelet.cpp:57:  Missing spaces around >  [whitespace/operators] [3]']
```
is output of notify_roslint_result.py currently.

**Comment by furushchev** [comment]
```xml
<file path="">
<warn commit="" line="">error_message</warn>
...
</file>
```

の形式が良いです。

**Comment by furushchev** [comment]
https://gist.github.com/609ff429beea596d3009

**Comment by wkentaro** [comment]
example output:

```xml
<roslint>
 <remote>
  <github owner="jsk-ros-pkg" repository="jsk_common" pullrequest="100" />
 </remote>
 <result>
  <file path="jsk_topic_tools/src/hz_measure_nodelet.cpp" commit="75c4a10">
   <error line="57">Missing spaces around >  [whitespace/operators] [3]</error>
  </file>
 </result>
</roslint>
```

**Comment by wkentaro** [comment]
@furushchev done.

**Comment by wkentaro** [comment]
above xml is expected output in https://github.com/jsk-ros-pkg/jsk_common/pull/1312

**Comment by garaemon** [comment]
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/trusty-travis-jsk-ros-pkg-jsk_travis-indigo-deb-true/1466/consoleFull

???

**Comment by garaemon** [comment]
too many y output

**Comment by garaemon** [comment]
I think this jenkins job kills other jobs

**Comment by wkentaro** [comment]
oops, sorry I will fix it.

**Comment by garaemon** [comment]
```
23:30:21 travis_time_end
23:30:23 Error response from daemon: database or disk is full
```

**Comment by garaemon** [comment]
```
$ df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda1       424G  263G  140G  66% /
udev             63G  4.0K   63G   1% /dev
tmpfs            13G  1.1M   13G   1% /run
none            5.0M     0  5.0M   0% /run/lock
none             63G   15M   63G   1% /run/shm
cgroup           63G     0   63G   0% /sys/fs/cgroup
/dev/sdb1       550G  146G  377G  28% /export/data1
/dev/sdc1       550G  550G     0 100% /export/data2
/dev/sdd1       550G  483G   40G  93% /export/data3
```

/dev/sdc is full

**Comment by garaemon** [comment]
@k-okada Could you cleanup /export/data2?

**Comment by wkentaro** [comment]
I'm sorry..

**Comment by garaemon** [comment]
I don't know why /export/data2/docker directory is so huge...

File: travis_jenkins.py [hunk 1]
```diff
@@ -267,6 +268,7 @@ def wait_for_building(name, number):
 CATKIN_PARALLEL_TEST_JOBS = %(CATKIN_PARALLEL_TEST_JOBS)s
 BUILD_PKGS       = %(BUILD_PKGS)s
 DOCKER_CONTAINER_NAME   = %(DOCKER_CONTAINER_NAME)s
+JENKINS_HOME            = %(JENKINS_HOME)s
```
Comment (wkentaro) [inline travis_jenkins.py]: @k-okada 
Could you please review this why this happen?
https://s3.amazonaws.com/archive.travis-ci.org/jobs/98149483/log.txt
```
JENKINS_HOME            = None

next build number is 1367
wait for trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true 1367
start building, wait for result....
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true/1367/ building.. True result... None
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true/1367/ building.. True result... None
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true/1367/ building.. True result... None
Started by user k-okada
Building on master in workspace /var/lib/jenkins/jobs/trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true/workspace
[workspace] $ /bin/sh -xe /tmp/hudson1311767791672845649.sh
+ set -x
+ set -e
+ env
JENKINS_HOME=/var/lib/jenkins
rvm_tmp_path=/usr/share/ruby-rvm/tmp
MAIL=/var/mail/jenkins
rvm_rubies_path=/usr/share/ruby-rvm/rubies
rvm_wrappers_path=/usr/share/ruby-rvm/wrappers
USER=jenkins
HUDSON_URL=http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/
SHLVL=1
NODE_LABELS=master
HOME=/var/lib/jenkins
```

```
+ source None/.token

/bin/bash: line 226: None/.token: No such file or directory

+ echo 'GitHub Token not found. skipping'
```

**Comment by wkentaro** [comment]
Closing with same comment: https://github.com/jsk-ros-pkg/jsk_common/pull/1312#issuecomment-223880444

**Pull request closed without merging**